Source/WebService/Blackjack.WebService/ClientApp/src/app/components/play-game/play-game.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="mainBody">
+  <div class="fundRelated">
+    <label>Your Balance: {{user.balance}}</label>
+    <label>Current wager: {{wager}}</label>
+    <div [class.hidden]="isPregameHidden" class="wagerBtns">
+      <button (click)="_addToWager(1)" class="gameBtn btn btn-info">$1</button>
+      <button (click)="_addToWager(5)" class="gameBtn btn btn-info">$5</button>
+      <button (click)="_addToWager(25)" class="gameBtn btn btn-info">$25</button>
+      <button (click)="_addToWager(100)" class="gameBtn btn btn-info">$100</button>
+      <button (click)="_addToWager(500)" class="gameBtn btn btn-info">$500</button>
+      <button (click)="_clearWager()" class="gameBtn btn btn-warning">Clear Wager</button>
+    </div>
+  </div>
+  <div [class.hidden]="isPregameHidden" class="pregame">
+    <label>Insert how many decks would you like to play with: </label>
+    <div class="pregameInput">
+      <input [(ngModel)]="deckQty" placeholder="deckQty" />
+      <button (click)="startGame()" class="gameBtn btn btn-success">Play!</button>
+    </div>
+  </div>
+  <div [class.hiden]="isPlayAreaHidden" class="playArea">
+    <label>{{player.status}}</label>
+    <label>{{dealer.status}}</label>
+    <label>{{nextMove}}</label>
+    <div>
```

Comment:
Empty div isn't doin anything here. Remove it before MTM

Reply:
This div contains the Hit and Stay Buttons unless I'm missing something?

Reply:
The div isn't doing anything though. It just has some elements in it. No style on it. No bearing on how anything renders

Reply:
It keeps the hit and stay button side by side rather than being affected by the column on the parent playArea div